### DISCUSSION TITLE
Incredible Idea

### DISCUSSION THREAD

**Discussion by JeremyLARDENOIS** [Pokemon-PythonRed/Pokemon-PythonRed#5] [General]
Hello,

I tested your project and i think that the idea is very cool ! The system of menu is very cool !

I find your project because you fork mine, if you want it's now translated :) 

Have a nice day :)

**Comment by riverpaper** [comment]
@JeremyLARDENOIS Oh that's really great, thanks for the feedback! Do you need any help with translations?

**Reply by JeremyLARDENOIS** [reply]
Hello, for the moment i correct all issues that I found. But if you want to correct my translations (i'm not a english native speaker), help or just test, you can, it will be with pleasure. Don't hesitate to ask if you need help with your project ;)

**Reply by riverpaper** [reply]
@JeremyLARDENOIS Thanks so much! I'll see what I can do.
